x-pack/plugins/lens/public/types.ts

```diff
@@ -608,6 +608,17 @@ export interface LensBrushEvent {
   data: TriggerContext<typeof SELECT_RANGE_TRIGGER>['data'];
 }
 
+export interface LensSortActionData {
```

Comment:
Can you move this type into the datatable visualization and make `LensEditEvent` a generic type for all kinds of payloads?